.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
```

Comment:
There's a option to trim trailing whitespace, which should make the pre-commit hook complain less often.